code/src/components/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./Header.css"
+
+const Header = () => {
+    return (
+        <header className="header-container">
+            <h1 className="title"> Friday Night Movies </h1>
+            </header>
+    );
+};
+
+export default Header;
```

Comment:
same here! (This file could be hidden or deleted since you don't seem to be using it in the final version?)